pages/docs/reference/faq.md

```diff
@@ -89,9 +89,11 @@ On the JVM side, the main build tools include [Gradle](/docs/reference/using-gra
 When targeting the JVM, Kotlin produces Java compatible bytecode. When targeting JavaScript, Kotlin transpiles to ES5.1 and generates
 code which is compatible with module systems including AMD and CommonJS. When targeting native, Kotlin will produce platform-specific code (via LLVM). 
 
-### Does Kotlin only target Java 6?
+### Which versions of JVM does Kotlin target?
 
-No. Kotlin lets you choose between generating Java 6 and Java 8 compatible bytecode. More optimal byte code may be generated for higher versions of the platform.
+Kotlin lets you choose the version of JVM for execution. By default, the Kotlin/JVM compiler produces Java 6 compatible bytecode.
+If you want to make use of optimizations available in newer versions of Java, you can explicitly specify the target Java version from 8 to 12.
```

Comment:
Note that we'll support JVM target 13 since 1.3.70, would be nice to update this again after release